ts/screens/profile/SecurityScreen.tsx

```diff
@@ -0,0 +1,153 @@
+    getFingerprintSettings().then(
+      biometryTypeOrUnsupportedReason => {
+        setIsFingerprintAvailable(
+          biometryTypeOrUnsupportedReason !== "UNAVAILABLE" &&
+            biometryTypeOrUnsupportedReason !== "NOT_ENROLLED"
+        );
+      },
+      _ => undefined
+    );
+  }, []);
+
+  // FIXME: Add alert if user refused IOS biometric permission on
+  // the first time the app is opened: https://pagopa.atlassian.net/browse/IA-67
+
+  const setBiometricPreference = (biometricPreference: boolean): void => {
+    if (biometricPreference) {
+      // if user asks to enable biometric then call enable action directly
+      setFingerprintPreference(biometricPreference);
+      return;
+    }
+    // if user asks to disable biometric recnognition is required to proceed
+    TouchID.authenticate(
+      I18n.t("identification.biometric.popup.title"),
+      authenticateConfig
+    )
+      .then(() => setFingerprintPreference(biometricPreference))
+      .catch((_: AuthenticationError) =>
+        // this toast will be show either if recognition fails (mismatch or user aborts)
+        // or if meanwhile user disables biometric recognition in OS settings
+        showToast(
+          I18n.t(
+            "profile.security.list.biometric_recognition.needed_to_disable"
+          ),
+          "danger"
+        )
+      );
+  };
+
+  const onPressBiometricRecognitionItem = () => {
+    if (isScreenReaderEnabled) {
+      setBiometricPreference(!isFingerprintEnabled);
+    }
+  };
+
+  return (
+    <TopScreenComponent
+      contextualHelpMarkdown={contextualHelpMarkdown}
+      faqCategories={["profile", "privacy", "authentication_SPID"]}
+      goBack
+    >
+      <ScreenContent
+        title={I18n.t("profile.security.title")}
+        subtitle={I18n.t("profile.security.subtitle")}
+      >
+        <List withContentLateralPadding>
+          {/* Ask for verification and reset unlock code */}
+          <ListItemComponent
+            title={I18n.t("identification.unlockCode.reset.button_short")}
+            subTitle={I18n.t("identification.unlockCode.reset.subtitle")}
+            onPress={requestIdentificationAndResetPin}
+            testID="reset-unlock-code"
+          />
+          {/* Enable/disable biometric recognition */}
+          {isFingerprintAvailable && (
+            <ListItemComponent
+              title={I18n.t(
+                "profile.security.list.biometric_recognition.title"
+              )}
+              subTitle={I18n.t(
+                "profile.security.list.biometric_recognition.subtitle"
+              )}
+              onSwitchValueChanged={setBiometricPreference}
+              switchValue={isFingerprintEnabled}
+              isLongPressEnabled
+              accessibilityState={{ checked: isFingerprintEnabled }}
+              accessibilityRole="switch"
+              onPress={onPressBiometricRecognitionItem}
+              testID="biometric-recognition"
+            />
```

Comment:
Since the `onSwitchValueChanged` and the `onPress` props should be executed alternatively based on the screen reader:
```suggestion
            <ListItemComponent
              title={I18n.t(
                "profile.security.list.biometric_recognition.title"
              )}
              subTitle={I18n.t(
                "profile.security.list.biometric_recognition.subtitle"
              )}
              onSwitchValueChanged={!isScreenReaderEnabled ? setBiometricPreference : undefined}
              switchValue={isFingerprintEnabled}
              isLongPressEnabled
              accessibilityState={{ checked: isFingerprintEnabled }}
              accessibilityRole="switch"
              onPress={isScreenReaderEnabled ? setBiometricPreference : undefined}
              testID="biometric-recognition"
            />
```
This because when the screen reader is **enabled** the `onPress` prop is executed if you press the whole item but the `onSwitchValueChanged` prop execute the same function with the opposite value.

While when the screen reader is **disabled** the value should change only when the user press on the switch (`onSwitchValueChanged` prop)